docs/core/compatibility/core-libraries/6.0/filestream-file-locks-unix.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Breaking change: FileStream locks files with shared lock on Unix"
+description: Learn about the .NET 6 breaking change where FileStream.Lock now locks files with a read lock when they're opened with read permissions on Unix.
```

Comment:
Should this `FileStream.Lock` mentioned in the description be wrapped by an xref or cref?

Reply:
No, it's just metadata so I don't think such links would resolve.